docs/quick-start.md

```diff
@@ -71,7 +72,7 @@ sudo ig daemon --host=tcp://0.0.0.0:1234
 
 Download the `gadgetctl` tools for MacOS
 ([amd64](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-darwin-amd64-v0.30.0.tar.gz),
-[arm64](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-darwin-arm64-v0.30.0.tar.gz)) or [windows](https://github.com/inspektor-gadget/inspektor-gadget/releases/latest/download/gadgetctl-windows-amd64-v0.30.0.tar.gz) and run the Gadget specifying the IP address of the Linux machine:
+[arm64](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-darwin-arm64-v0.30.0.tar.gz)) or [windows](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-windows-amd64-v0.30.0.tar.gz) and run the Gadget specifying the IP address of the Linux machine:
```

Comment:
The only way I see is to provide the commands to do it manually as we do for https://github.com/inspektor-gadget/inspektor-gadget?tab=readme-ov-file#install-locally. It would be easy for MacOS, but what about Windows? Should we provide powershell commands? 😅

Reply:
IMO it is good enough to just give the generic link:
https://github.com/inspektor-gadget/inspektor-gadget/releases
rather than risking a out of date link.

We cannot update files in `docs/` with a bot creating a PR after a release because the documentation website is created with the release tag, so the PR would be too late.

But we can create an automatic PR for README.md.

Reply:
btw, I'd like to keep the quick start and the readme as aligned as possible, otherwise it'll be so difficult to maintain if they diverge a lot.